compiler/src/iree/compiler/Codegen/LLVMGPU/ConvertToROCDL.cpp

```diff
@@ -97,6 +97,8 @@ struct ConvertToROCDLPass final
       return signalPassFailure();
     }
     bool use32BitIndices = clROCMIndexingBits == 32;
+    StringRef chipset = getGPUTargetAttr(m).getArch();
+    FailureOr<amdgpu::Chipset> maybeChipset = amdgpu::Chipset::parse(chipset);
```

Comment:
We should emit an error if failed, since it's required below and there's no good default value